src/code_generator_templates/ts/ts_gateway_layer_template.ts

```diff
@@ -0,0 +1,85 @@
+
+
+
+export function getTSOrchestratorCodeTemplate(featureClassName: string, snakeCaseFeatureName: string): string {
+    const template = 
+    
+    `
+    import { container } from "../../../inversify.config";
```

Comment:
This path might be wrong right?

Reply:
Im assuming the folder would always be created inside the src file 
So it shoudn't be wrong